### ISSUE TITLE
where is segmentation task?

### ISSUE THREAD

**Issue by dreamkily** [leekyungmoon/XDED#1]

**Comment by Jin-huihuang** [comment]
how to apply on segmentation task？

**Comment by leekyungmoon** [comment]
Thank you for reaching out.

Unfortunately, we don’t have a complete semantic segmentation code in this repository.
However, we provide the XDED loss function applicable for semantic segmentation ([here](https://github.com/leekyungmoon/XDED/blob/main/dassl/modeling/ops/cross_entropy.py#L44-L78)). Please refer to the codes in the attached link and use it as you intended.
As stated in the paper, the ensembles are constructed by simply averaging all the logits from the pixels of the same GT class in a mini-batch.

Although this might not cover all your needs, we hope the provided code aids your research.
Thank you.
